common/util/web_controller/autoware_web_controller/package.xml

```diff
@@ -40,7 +40,8 @@
   <!--   <test_depend>gtest</test_depend> -->
   <!-- Use doc_depend for packages you need only for building documentation: -->
   <!--   <doc_depend>doxygen</doc_depend> -->
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
```

Comment:
We can remove this one, as it'll work with just `ament_cmake`
```suggestion
```

Reply:
this is fixed